### DISCUSSION TITLE
365 Search and Intelligence - answers in search results

### DISCUSSION THREAD

**Discussion by stuartcporter** [microsoft-search/pnp-modern-search#2077] [Q&A] [answered]
HI 
has anybody managed to get answers (acronyms, locations q&a etc) from search & intelligence admin center to display when using the microsoft search data source - these answers are showing when i search from bing or the microsoft search from the sharepoint.aspx page in sharepoint online. however they do not show when i change my search data source to microsoft search ( which is what i would expect to have these)

any help  - or clarification that these items are not currently included would be appreciated.
thanks
stuart

**Comment by wobba** [comment, marked as answer]
We haven't implemented support for it yet, and docs don't mention how to request them either from what I have seen. Only admin API docs.

**Reply by stuartcporter** [reply]
Thanks for the answer - be interesting if and when microsoft publish the details

**Reply by wobba** [reply]
As soon as we/I have the details we will add the functionality.
